game-core/src/main/java/games/strategy/engine/lobby/server/GameDescription.java

```diff
@@ -256,7 +251,6 @@ public void readExternal(final ObjectInput in) throws IOException {
     passworded = in.readBoolean();
     engineVersion = in.readUTF();
     gameVersion = in.readUTF();
-    botSupportEmail = in.readUTF();
```

Comment:
Since you're writing out an empty string for this field in `writeExternal()` (presumably to preserve lobby compatibility), do you also need to read (and discard) a string here for the same reason?  Otherwise the stream may become unsynchronized if multiple `GameDescription` instances have been serialized.